example.yaml

```diff
@@ -180,4 +178,30 @@ v6:
                         value:
                             - 2001:db8::1
                             - 2001:db8::2
-        
+
+# DDNS config
+# This section is optional, if not included, no DDNS updates will
+# be sent
+ddns:
+    # send updates. If the ddns section header is defined, enable_updates defaults to true
+    enable_updates: true
+    # default false. whether to override the client update FQDN flags
+    override_client_updates: false
+    # default false. whether to override the no update FQDN flags
+    override_no_updates: false
+    # list of forward DNS servers
+    # selects based on FQDN longest match
+    forward: 
+       - name: "example.com."
+         key: ""
```

Comment:
Empty key name looks weird. If intended, elaborate in a comment?

Reply:
Yes, I can remove that from the example.yaml. The `key` field is in fact optional so it doesn't need to be present